src/sensors/CyclePowerData.cpp

```diff
@@ -10,20 +10,15 @@
 
 bool CyclePowerData::hasHeartRate() { return false; }
 
-bool CyclePowerData::hasCadence() { return this->cadence > 0 || this->missedReadingCount < 3; }
+bool CyclePowerData::hasCadence() { return !isnan(this->cadence); }
```

Comment:
Logically, I think this is good.  The value starts as NAN and once we see data it will be set to >= 0.  `missedReadingCount` will still prevent dropouts.